### DISCUSSION TITLE
Remove icon space when no icon present

### DISCUSSION THREAD

**Discussion by ghost** [davatorium/rofi#1848] [Q&A]
![Screenshot_2023-06-02_14-47-18](https://github.com/davatorium/rofi/assets/135234789/094c22f7-85fe-4afe-a07b-8d155e563080)

In the screenshot you can see that the entries without an icon are still indented as if they had one. Is there a way to disable this in the theme config?

**Comment by DaveDavenport** [comment]
I don't think that is currently possible.
